release-notes.txt

```diff
@@ -1,3 +1,761 @@
+RIOT-2022.07 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release
+==================
+
+The 2022.07 release includes many changes, some highlights of which are:
+
+### GPIO Low-level drivers
+
+A new API, `gpio_ll`, has been introduced for highly efficient access to GPIO
+pins grouped in ports.
+
+The API is limited to internal use for the time being, but still paves the way
+towards using port multipliers like local GPIO pins, and for improved
+bit-banging implementations (e.g. around WS281x-style LEDs).
+
+### Improved serial port selection
+
+The automatic selection of serial ports has been improved by considering more
+metadata. When using multiple board of different kinds simultaneously, passing
```

Comment:
```suggestion
metadata. When using multiple boards of different kinds simultaneously, passing
```

Reply:
Applied together with other changes on that paragraph as ef4351e9b70ef3cae72cfcfea8beabd00e230011